metadata.sql

```diff
@@ -1,3 +1,4 @@
-CREATE TABLE FileMap(uuid text, server text);
-CREATE TABLE Server(server text);
+CREATE TABLE IF NOT EXISTS Stats(uuid text, connections int);
```

Comment:
Hm. Could we rename this table to something a bit more descriptive? Like `Connections` or something else?